src/global_env.cpp

```diff
@@ -120,6 +124,9 @@ void GlobalEnv::visit_children(Visitor &visitor) {
         visitor.visit(m_Encodings[i]);
     for (auto pair : m_files)
         visitor.visit(pair.first);
+    for (auto pair : m_interned_strings)
+        for (size_t i = 0; i < pair.size; i++)
+            visitor.visit(pair.strs[i]);
```

Comment:
```suggestion
    for (auto set : m_interned_strings)
        for (size_t i = 0; i < set.size; i++)
            visitor.visit(set.strs[i]);
```

I like `set` or `span` here rather than `pair`, since pair implies two.